src/guide/introduction.md

```diff
@@ -32,6 +32,7 @@ Here is a minimal example:
 <div class="options-api">
 
 ```js
+<script>
```

Comment:
Thank you for your suggestion! However, we cannot simply put `<script>` tag in `js` code block. We'd need to have `vue-html` block (we can merge it with the next one) and then put the tag there:

```vue-html
<script>
 // script part
 </script>

<template>
  // template part
</template>